packages/devtools_app/lib/src/routing.dart

```diff
@@ -0,0 +1,161 @@
+// Copyright 2020 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:collection';
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+const appSizePageId = 'app-size';
+const homePageId = '';
+const snapshotPageId = 'snapshot';
+
+/// Represents a Page/route for a DevTools screen.
+class DevToolsRouteConfiguration {
+  DevToolsRouteConfiguration(this.page, this.args);
+
+  final String page;
+  final Map<String, String> args;
+}
+
+/// Converts between structured DevToolsRouteConfiguration (our internal data
+/// for pages/routing) and RouteInformation (generic data that can be persisted
```

Comment:
nit: put DevToolsRouteConfiguration and RouteInformation in brackets []

Reply:
Done!